jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -124,22 +124,10 @@
 	
 	static final int PUBLIC_FINAL_NATIVE = Modifier.PUBLIC | Modifier.FINAL | Modifier.NATIVE | 0x1000 /* Synthetic */;
 	
-	private static final class UnsafeGetter {
-		public static final Unsafe myUnsafe = Unsafe.getUnsafe();
-	}
+	static final Unsafe _unsafe = Unsafe.getUnsafe();
```

Comment:
The convention is to use uppercase only when naming `static final` fields.

Reply:
Also, J9 JCL doesn't used a leading `_` on member variables.  Please remove this so that the code matches the existing conventions.